src/vnsw/agent/contrail/SConscript

```diff
@@ -104,8 +104,7 @@ if sys.platform.startswith('freebsd'):
 elif sys.platform.startswith('linux'):
     platform_dependent = 'linux/pkt0_interface.cc'
 elif sys.platform.startswith('win'):
-    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib']
-
+    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib', env.RES('windows/resource.rc')]
```

Comment:
Sorry I misunderstood your original question.  Yes, I just tried changing only resource.h. If you change resource.h, .rc is recompiled and the exe is re-linked.